samples/basic/blinky/README.rst

```diff
@@ -1,7 +1,8 @@
-.. _blinky-sample:
+.. zephyr:code-sample:: blinky-sample
+   :name: Blinky
+   :doxygen-groups: gpio_interface
```

Comment:
I had the exact same thought so, yes, agreed. 
In fact this field could even automagically let folks reference different "classes" of things, and not just doxygen groups, ex. ``:relevant-api: gpio_interface :c:func:`gpio_pin_toggle_dt` `` but this might just lead to clutter and too much granularity (although I do like the idea of potentially referencing a particular API call if a sample just happens to be the perfect showcase for it?)

Reply:
@gmarull what do you think?